src/plugins/blocks/blocks.js

```diff
@@ -238,6 +245,9 @@ export async function decorate(container, data) {
 
     // Show toast
     container.dispatchEvent(new CustomEvent('Toast', { detail: { message: 'Copied Block' } }));
+
+    // Track block copy event
+    sampleRUM('blockcopied');
```

Comment:
```suggestion
    sampleRUM('library:blockcopied');
```

Why is this happening twice?

Reply:
There are two places in the UI where you can copy the block. The one on the left allows you to copy the block without loading the preview.

![Screenshot 2023-06-12 at 5 01 20 PM](https://github.com/adobe/franklin-sidekick-library/assets/3231084/5892e35e-b4a8-4962-9014-272d0ab3c166)